### ISSUE TITLE
NTR: Winter (cold weather) onset (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7835]
New term request Winter (cold weather) onset (samant_101217013440): Cooler; lessening in warm weather  UMLS: 2039358

**Comment by pnrobinson** [comment]
@MickeySegal
analogous question as for https://github.com/obophenotype/human-phenotype-ontology/issues/7836

**Comment by MickeySegal** [comment]
No, this is for infections that spread or occur in winter, e.g., bacterial meningitis, viral bronchiolitis.

**Comment by pnrobinson** [comment]
@MickeySegal 
This term is a little difficult because it would be hard to keep users from applying the term to any disease that happens to occur in winter or summer. I think it is probably better to put a term such as **History of onset in winter**, so that it is clear that the term refers to one patient rather than being a characteristic of a disease.
Throughts?

**Comment by MickeySegal** [comment]
That is fine.  In my pediatrics internship we did 2 rotations through infectious disease, one to get the summer infections and one to get the winter infections.  Knowing the time of year made a huge difference to the diagnostic guesses.

In our software such findings are suggested by the Usefulness algorithm when appropriate.

**Comment by pnrobinson** [comment]
@MickeySegal 
Sorry for the delay.
Adding as 
label:History of disease onset in winter
id: HP:0020008
def: The initial manifestation of a disease or symptom occurring during the winter season.
